symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/datafeed/impl/AbstractDatafeedService.java

```diff
@@ -86,117 +86,117 @@ protected void handleV4EventList(List<V4Event> events) throws ApiException {
             if (this.isSelfGeneratedEvent(event)) {
                 continue;
             }
-            for (DatafeedEventListener listener : listeners) {
+            for (RealTimeEventListener listener : listeners) {
                 switch (event.getType()) {
```

Comment:
We could try to make this part of code a bit more generic by using an `enum` instead of a `switch` (cf. https://github.com/SymphonyOSF/symphony-sdk-java-v2/blob/master/symphony-sdk-api/src/main/java/com/symphony/sdk/api/datafeed/DatafeedEventType.java). We can discuss about that later.

Reply:
+1